internal/v2/certverifier/certverifier_test.go

```diff
@@ -55,15 +57,21 @@ var (
 	serverLeafDERCert []byte
 )
 
-func startFakeS2Av2Server(wg *sync.WaitGroup) (stop func(), address string, err error) {
+func startFakeS2Av2Server(wg *sync.WaitGroup, enableServerAuthorizationpolicy bool) (stop func(), address string, err error) {
```

Comment:
nit: Can we rename the variable to `enableServerAuthorizationPolicyCheck`? Note that adding the "Check" suffix and also fixing the typo in camel-case.

Reply:
Done.